.gitpod.dockerfile

```diff
@@ -0,0 +1 @@
+FROM gitpod/workspace-java-17:2023-07-20-19-56-24
```

Comment:
```suggestion
FROM gitpod/workspace-java-17
```

Reply:
Let's make it pull the latest image

Reply:
[Dockerfile Linter](https://hadolint.github.io/hadolint/) complains about [not specifying the exact version](https://github.com/hadolint/hadolint/wiki/DL3006). From my experience, the `Dependabot` does a good job in updating these.